Robus/inc/robus_struct.h

```diff
@@ -149,6 +149,7 @@ typedef enum
     // protocol level command
     WRITE_NODE_ID,   /*!< Get and save a new given node ID. */
     RESET_DETECTION, /*!< Reset detection*/
+    END_DETECTION,   /*!< Detect the end of a detection*/
```

Comment:
If we have `END_DETECTION` I think this is better to have `START_DETECTION` instead of `RESET_DETECTION`